.gitignore

```diff
@@ -11,3 +11,5 @@ sqlnet.log
 Gemfile.lock
 *.zip
 .vagrant
+.idea
+spec/support/custom_config.rb
```

Comment:
Is it possible to fix that "No new line at end of file" warning? (If it's hard to do on Windows - let me know, I can push a fix for that).